cmd/root/sweet.go

```diff
@@ -467,15 +467,17 @@ func (m exerciseModel) exerciseTextView() (s string) {
 func (m exerciseModel) View() (s string) {
 	if !m.finished() {
 		currKeyI := len(m.typedText)
-		currKey := string(m.exercise.text[currKeyI])
+		currKey := m.exercise.text[currKeyI]
+		margin := 5
 		s += "\n"
 		s += m.exerciseNameView()
 		s += "\n\n"
 		s += m.exerciseTextView()
 		s += "\n"
 		s += "keymap:\n\n"
-		s += qwerty.render(currKey)
-		s += "\n\n"
+		s += qwerty.render(string(currKey))
```

Comment:
Keeping fingers separate from the keymap
- decouples the finger view from the keymap view, for ease of development

Keeping finger and the keymap closer
- makes sense, since the placement of the fingers is dependent on the position of the keymap. Fingers should be aligned to specific keys when using the keyboard.
I think I found a decent balance for now. will show in an update.